federation/source-config-handlers/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -28,6 +28,7 @@
           class="org.codice.ddf.admin.sources.csw.CswSourceConfigurationHandler">
         <argument ref="sourceConfigurationHandler"/>
         <argument ref="configuratorFactory"/>
+        <argument ref="securityManager"/>
```

Comment:
We should retrieve the securityManager through a Configurator instead. This will allow us to mock out services in one place in the future. This should also reduce the additional parameters made from passing the securityManager around since nearly all of them are already passing the configuratorFactory.